examples/counter/Forc.lock

```diff
@@ -1,13 +1,11 @@
 [[package]]
 name = 'core'
-source = 'git+https://github.com/FuelLabs/sway-lib-core?branch=master#082bc8eb0616586ac0000825bfe6e9d47b0c713f'
 dependencies = []
 
 [[package]]
 name = 'counter'
-dependencies = ['std git+https://github.com/FuelLabs/sway-lib-std?branch=master#3884e9bbadb7c2567700dae847017366629241d3']
+dependencies = ['std']
```

Comment:
I'm surprised a path isn't included in the lockfile???

Reply:
No `path` is necessary in the lock file, as the local path is declared under `Forc.toml` and there's no ambiguity around which instance of the package to select. This is unlike remote git sources where a single git reference might be associated with a list of possible commits, and we need to be able to reliably fetch the same commit even without control over the remote repository.